ipatests/pytest_ipa/integration/create_bridge.py

```diff
@@ -20,6 +21,19 @@ def setup_scim_server(host, version="main"):
     host.run_command(["python", "./prepare_sssd.py"],
                      cwd=f"{dir}/src/install")
 
+    # Get keytab for scim bridge service
+    master = host.domain.hosts_by_role("master")[0].hostname
+    princ = f"admin@{host.domain.realm}"
+    ktfile = "/root/scim.keytab"
+    sendpass = f"{password}\n{password}"
+    tasks.kdestroy_all(host)
+    tasks.kinit_admin(host)
+    host.run_command(["ipa-getkeytab", "-s", master, "-p", princ,
+                      "-P", "-k", ktfile], stdin_text=sendpass)
+    host.run_command(["klist", "-k", "-t", ktfile])
+    time.sleep(1)
```

Comment:
No need to add sleep if you are running next step repeatedly

Reply:
I removed the sleep(), the klist, and switched run_repeatedly back to just a run_command.